### DISCUSSION TITLE
［灌水］今天接着整点大家想看的东西啊

### DISCUSSION THREAD

**Discussion by fzdqz** [share121/inter-knot#5320] [灌水]
![illust_121226780_20240918_003140](https://github.com/user-attachments/assets/facca4dd-917e-4fbc-8807-2ac5495b1b1e)

**Comment by fzdqz** [comment]
https://b23.tv/Lrnv6ZR
点进去有教程∠( ᐛ 」∠)_

**Comment by KELERSAMA** [comment]
传奇钓鱼佬钓不上鱼了 
回家吧 快下雨了 
雨天路滑 小心电瓶翻车

**Comment by heshuikaya** [comment]
图片仅供参考是吧ԅ(¯㉨¯ԅ)

**Comment by aino2333** [comment]
什么图片仅供参考（

**Comment by 210flight** [comment]
《图片仅供参考》

**Comment by cpu644** [comment]
大伙都看过就没意思了
